lib/sensu/server/process.rb

```diff
@@ -354,7 +359,7 @@ def process_check_result(result)
               result[:check]
             end
             aggregate_check_result(result) if check[:aggregate]
-            store_check_result(client, check) do
+            store_check_result(client, check, result) do
```

Comment:
the `result` arg should not be required, since the provided `client` and `check` should have all of the data, no?